src/frontends/pytorch/src/op/cross.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/roll.hpp"
+#include "openvino/op/subtract.hpp"
+#include "openvino/op/unsqueeze.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+namespace {
+Output<Node> translate_cross_base(const NodeContext& context, Output<Node> self, Output<Node> other, Output<Node> dim) {
+    auto const_1 = context.mark_node(v0::Constant::create(element::i32, Shape{1}, {1}));
```

Comment:
Don't we need to align types for self and other?

Reply:
RuntimeError: expected scalar type Float but found Double happens in torchscript on inference stage, but tracing successfully passed

Reply:
type alignment added